cli/default-reporter/src/reportError.ts

```diff
@@ -21,7 +21,7 @@ const colorPath = chalk.gray
 
 export function reportError (logObj: Log, config?: Config) {
   const errorInfo = getErrorInfo(logObj, config)
-  let output = formatErrorSummary(errorInfo.title, logObj['err']['code'])
+  let output = formatErrorSummary(errorInfo.title, (logObj as LogObjWithPossibleError).err?.code)
```

Comment:
I would normally add a test when fixing a bug, but I think in this case this bug happened as a result of intentionally bypassing TypeScript compilation checks. The `LogObjWithPossibleError` interface here should make this a bit safer in the future.